bin/travis-build.bash

```diff
@@ -20,7 +20,7 @@ then
 	cd ckan
 	git checkout ckan-2.8.4
 	# apply required patch
-	curl -L https://raw.githubusercontent.com/GSA/catalog.data.gov/75_fix_save_harvest_source/ckan/patches/ckan/unflattern_indexerror.patch > 1.patch
+	curl -L https://raw.githubusercontent.com/GSA/catalog.data.gov/master/ckan/patches/ckan/unflattern_indexerror.patch > 1.patch
```

Comment:
Since applying patches is already part of the docker build, maybe we should just build and run docker in CI? That would bring better parity between CI and local development, too.

How hard would this be?

Reply:
We are doing this in the local docker build tests